src/main/java/com/mcmoddev/bot/command/CommandXY.java

```diff
@@ -0,0 +1,34 @@
+package com.mcmoddev.bot.command;
+
+import com.mcmoddev.bot.util.Utilities;
+import sx.blah.discord.handle.obj.IMessage;
+import sx.blah.discord.util.EmbedBuilder;
+
+public class CommandXY implements Command
+{
+    @Override
+    public void processCommand(IMessage message, String[] params)
+    {
+        final StringBuilder builder = new StringBuilder();
+        final EmbedBuilder embed = new EmbedBuilder();
+
+        builder.append(
```

Comment:
The whole point of a StringBuilder is to not have chained string addition, like so: 
```
"first " + "second"
```